packages/core/src/indexing/postgres/postgres-index-api.ts

```diff
@@ -72,6 +72,7 @@ export class PostgresIndexApi implements DatabaseIndexApi {
       .insert(indexedData)
       .onConflict('stream_id')
       .merge({
+        stream_content: indexingArgs.streamContent,
```

Comment:
good catch!  This does indeed seem like a bug to me.  I think we probably also want to add `tip` into this merge to be updated as well, here and in the sqlite version.  Would you mind adding that change in as well?

Reply:
Of course. I've added the tip.